gtsfm/two_view_estimator.py

```diff
@@ -28,7 +33,20 @@
 pil_logger = logging.getLogger("PIL")
 pil_logger.setLevel(logging.INFO)
 
-EPSILON = 1e-6
+PRE_BA_REPORT_TAG = "PRE_BA_2VIEW_REPORT"
+POST_BA_REPORT_TAG = "POST_BA_2VIEW_REPORT"
+POST_ISP_REPORT_TAG = "POST_INLIER_SUPPORT_PROCESSOR_2VIEW_REPORT"
+POST_CYCLE_CONSISTENT_REPORT_TAG = "POST_CYCLE_CONSISTENT_2VIEW_REPORT"
+
+
+DATA_ASSOC_REPROJ_ERROR_THRESH = 3
+BA_OUTPUT_REPROJ_ERROR_THRESH = 100
+DATA_ASSOC_2VIEW = DataAssociation(
+    reproj_error_thresh=DATA_ASSOC_REPROJ_ERROR_THRESH, min_track_len=2, mode=TriangulationParam.NO_RANSAC
+)
+BUNDLE_ADJUST_2VIEW = BundleAdjustmentOptimizer(
+    output_reproj_error_thresh=BA_OUTPUT_REPROJ_ERROR_THRESH, robust_measurement_noise=True, max_iterations=10
+)  # we dont care about output error threshold as we do not access the tracks of 2-view BA, which the threshold affects.
```

Comment:
Should we ass these to the config?

Reply:
yeah, line 42 and 43 should be loaded from the config

Reply:
43 is inconsequential. I am basically ignoring the optimized points for now. 

Having 3 in a config is not so straightforward. I think once we have a batch triangulation API, we will just use that instead of using the data association module. So i think we can avoid this effort for now.

Reply:
I think we should at least be able to turn 2-view BA on or off from the config, especially due to the greatly inflated runtimes.

Reply:
We dont have that much config anymore. Let me know if you still think we need the configs.

Reply:
I still think we should be able to turn two-view BA on or off from the config at least, and also maybe change the max iterations.

Reply:
Every blue box should have a a config, right? That are sections in some main config file?